ws-nextjs-app/pages/[service]/live/[id]/live.test.tsx

```diff
@@ -62,6 +64,44 @@ describe('Live Page', () => {
     expect(screen.getByText('Pidgin test 2')).toBeInTheDocument();
   });
 
+  it('should use the title value from the data response as the page title', async () => {
+    await act(async () => {
+      render(<Live pageData={mockPageData} />, { service: 'pidgin' });
+    });
+
+    const { title: helmetTitle } = Helmet.peek();
+
+    expect(helmetTitle).toEqual(`${mockPageData.title} - BBC News Pidgin`);
+  });
+
+  it('should use the title value combined with the pagination value as the page title', async () => {
+    const paginatedData = {
+      ...mockPageData,
+      liveTextStream: {
+        content: {
+          data: {
+            results: [],
+            page: {
+              index: 2,
+              total: 3,
+            },
+          },
+        },
+        contributors: 'Not a random dude',
+      },
+    };
+
+    await act(async () => {
+      render(<Live pageData={paginatedData} />, { service: 'pidgin' });
+    });
+
+    const { title: helmetTitle } = Helmet.peek();
+
+    expect(helmetTitle).toEqual(
+      `${mockPageData.title}, Page 2 of 3 - BBC News Pidgin`,
```

Comment:
I guess we have translations for 'Page 2 of 3' etc from the topic page work?

Reply:
Yea we do indeed. Just so happens that the Pidgin translations are `Page`: https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/pidgin.ts#L51, other services are translated where appropriate though.